.gitlab-ci.yml

```diff
@@ -2583,7 +2583,31 @@ deploy_puppy:
   script:
     - $S3_CP_CMD $S3_ARTIFACTS_URI/puppy/agent ./agent
     - export PACKAGE_VERSION=$(inv agent.version --url-safe --major-version 7)
-    - aws s3 cp --region us-east-1 ./agent $S3_DSD6_URI/puppy/agent-$PACKAGE_VERSION --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
+    - aws s3 cp --region us-east-1 ./agent $S3_DSD6_URI/linux/puppy/agent-$PACKAGE_VERSION --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
+
+# deploy agent windows zip to the staging bucket, currently used for cloudfoundry bosh
+deploy_datadog_agent_windows_zip:
+  <<: *run_when_triggered_on_tag_7
+  stage: deploy7
+  needs: [ "windows_msi_x64-a7"]
```

Comment:
Overall naming in our pipeline is quite confusing. This is another example: the `windows_msi_x64-a7` job generates both an MSI and a ZIP.